FirebaseFunctions/Sources/Internal/FunctionsContext.swift

```diff
@@ -35,7 +35,9 @@ internal class FunctionsContextProvider: NSObject {
   private var messaging: MessagingInterop?
   private var appCheck: AppCheckInterop?
 
-  init(auth: AuthInterop?, messaging: MessagingInterop?, appCheck: AppCheckInterop?) {
+  // The 'public' here is weird. It is sometimes needed for `@testable import FirebaseFunctions` to
+  // successfully access this 'init', but it depends upon the build configuration. Xcode 13.2.1.
+  public init(auth: AuthInterop?, messaging: MessagingInterop?, appCheck: AppCheckInterop?) {
```

Comment:
While putting `public` here fixed it locally for me, it doesn't on CI. It seems like there is a race in the Swift build system and the processing of `testable`.

Any ideas?